integration/emr/alluxio-emr.sh

```diff
@@ -170,13 +171,15 @@ main() {
   print_help() {
     local -r USAGE=$(cat <<USAGE_END
 
-Usage: alluxio-emr.sh <root-ufs-uri> [-b <backup_uri>]
-                             [ -d <alluxio-download-uri>]
-                             [-f <file_uri>]
-                             [-i <journal_backup_uri>]
-                             [-n <storage percentage>]
-                             [-p <delimited_properties>]
-                             [-s <property_delimiter>]
+Usage: alluxio-emr.sh [-u <root-ufs-uri>]
```

Comment:
@yuzhu why introduce a new flag? we are breaking backwards compatibility here

Reply:
the alternative is to use empty string to represent "use local hdfs as root ufs" 
I am fine with that too.

Reply:
is that possible? would bash read this correctly? `alluxio-emr.sh "" -a someArg -b someArg ...`